src/types/types.ts

```diff
@@ -293,7 +293,20 @@ export type FakeCommitHash =
   | '9d50665254899d8431813bdc04178e6006ce6d59'
   | 'a998c42399a8fcea623690bf65bef49de20535b4';
 
-export type UserCredentials = Record<
-  string,
-  { expires: string; credentials: { clientId: string; accessToken: string } }
->;
+export type CredentialsResponse = {
+  expires: string;
+  credentials: { clientId: string; accessToken: string };
+};
+
+export type UserCredentials = Record<string, CredentialsResponse>;
+
+export type TokenBearerResponse = {
+  access_token: string;
+  token_type: 'Bearer';
+};
+
+export type TokenBearer = Record<string, TokenBearerResponse>;
+
+export type TokenResponse = TokenBearerResponse;
+
+export type UserToken = Record<string, TokenResponse>;
```

Comment:
it looks like you have the same types twice:
TokenBearerResponse == TokenResponse
TokenBearer == UserToken

Please pick just one pair (I'd suggest TokenBearerResponse / TokenBearer)

Optional nit: it would be good to move the directionary types to `credentials-storage.ts` because they're used only there, they're not global.

Optional nit: Rename:

CredentialsResponse => UserCredentials
UserCredentials => UserCredentialsDictionary
TokenBearerResponse => TokenBearer
TokenBearer => TokenBearerDictionary

(with the Dictionary types in credentials-storage.ts as suggested above, and the other types kept here)

Reply:
Thank you! I was confused about how to keep them.